locales/en/common.json

```diff
@@ -9,6 +9,7 @@
   "HOME_LINK": "Home",
   "COLLECTION_LINK": "Collections",
   "CREATE_LINK": "Create Nengajo",
+  "MYPAGE": "マイページ",
```

Comment:
I would like to suggest "Collection" instead of "my page/マイページ" because:

- "My page" is not commonly used in English, and it is nice to use similar words if we can
- Same as on OpenSea

Reply:
@alecrem 
Thank you for suggestion
I changed /mypage => /collection and remove mypage

Reply:
Thank you! I forgot we had "Collection" already.